### DISCUSSION TITLE
How can i increase the line-height ?

### DISCUSSION THREAD

**Discussion by biomathcode** [outline/rich-markdown-editor#374] [Q&A] [answered]
The line height of h1 and h2 are small and make the text look back. I have the text overlapping and also while highlight. 
![Capture](https://user-images.githubusercontent.com/29139349/106597938-13a79800-657d-11eb-9f7e-b397acf67b70.PNG)

**Comment by biomathcode** [comment, marked as answer]
Okay i got it i can use a simple style.css. I am stupid...

**Comment by theodorechu** [comment]
I think this happens because the default line-height is `1.7em` as set in [index.tsx](https://github.com/outline/rich-markdown-editor/blob/5ec7405a6a0458d47bcc839c976d173086cc66a3/src/index.tsx#L655). To prevent the headers from overlapping, the line-height could be changed to `1.7` as discussed on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values). If you don't want the line-height to be so big (`1.7` is quite large for headers), then you can add additional CSS to make the line-height for `h1`, `h2`, and `h3` be `normal`.

**Reply by biomathcode** [reply]
I added additional css to make the line-height different  for h1, h2, and h3.
